types/react-i18next/v4/translate.d.ts

```diff
@@ -11,5 +12,14 @@ export interface TranslateOptions {
     i18n?: i18n;
 }
 
+// Diff / Omit taken from https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766
+export type Omit<T, K extends keyof T> = Pick<T, ({ [P in keyof T]: P } & { [P in K]: never } & { [x: string]: never, [x: number]: never })[keyof T]>;
```

Comment:
I know this is backported from the main files, but as this is TS 2.8 we should be able to just written this as `Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>`.